scenarioo-server/src/test/java/org/scenarioo/business/diffViewer/comparator/ScreenshotComparatorGraphicsMagickTest.java

```diff
@@ -48,6 +50,23 @@ public void setUpClass() throws IOException {
 			getComparisonConfiguration());
 	}
 
+	// Avoids massive logging which is expected
+	private Enumeration allAppenders;
+
+	@Before
+	public void removeAppenders() {
+		allAppenders = Logger.getRootLogger().getAllAppenders();
+		Logger.getRootLogger().removeAllAppenders();
+		Logger.getRootLogger().addAppender(new NullAppender());
+	}
+
+	@After
+	public void restoreAppenders() {
+		for (Object appender : Collections.list(allAppenders)) {
+			Logger.getRootLogger().addAppender((Appender) appender);
+		}
+	}
```

Comment:
Können wir nicht einfach sinnvolle logs machen und Exceptions die wir sowieso handeln besser nicht logen??
Finde es etwas komisch wenn nicht gar problematisch (für späteres Debuging von failenden Tests) die Logs in einem Test extra aszuschalten (und dann noch für alle Tests und nicht nur für einzelne, welche irgendwelche Fehler-Situationen testen).
Was im Test stört, stört auch später, oder etwa nicht? Warum also nicht gleich die Komponente so anpassen, dass sie keine unnötigen Log-Outputs fabriziert.